src/main/java/org/team340/robot/subsystems/Intake.java

```diff
@@ -0,0 +1,58 @@
+package org.team340.robot.subsystems;
+
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import com.ctre.phoenix6.hardware.TalonFX;
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj2.command.Command;
+import java.util.function.DoubleSupplier;
+import org.team340.lib.util.command.GRRSubsystem;
+import org.team340.lib.util.vendors.PhoenixUtil;
+import org.team340.robot.Constants.RobotMap;
+
+public class Intake extends GRRSubsystem {
+
+    final double kIntakingSpeed = 0.0;
```

Comment:
```suggestion
    private static final TunableDouble kIntakingSpeed = Tunable.doubleValue("intake/kIntakingSpeed", 0.0);
```
Constants should be static and tunable